From a606908b31b9b058c838f0c814c5dda4487afba3 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Thu, 7 Apr 2011 09:22:14 +0200
Subject: [PATCH] i18n: make sure to use the "bluedevil" catalog everywhere

... otherwise some parts (like the kcm) will not get any translation from the single "bluedevil" catalog
---
 src/actionplugins/actionplugin.h   |    2 +-
 src/kcmodule/bluedeviladapters.cpp |    2 +-
 src/kcmodule/bluedevildevices.cpp  |    2 +-
 src/kcmodule/bluedeviltransfer.cpp |    2 +-
 src/kio/bluetooth/kiobluetooth.cpp |    2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/actionplugins/actionplugin.h b/src/actionplugins/actionplugin.h
index 78012a1..145b3d0 100644
--- a/src/actionplugins/actionplugin.h
+++ b/src/actionplugins/actionplugin.h
@@ -27,7 +27,7 @@
 
 #define BLUEDEVILACTION_PLUGIN_EXPORT( c ) \
   K_PLUGIN_FACTORY( BlueDevilAction, registerPlugin< c >(); ) \
-  K_EXPORT_PLUGIN( BlueDevilAction("c") )
+  K_EXPORT_PLUGIN( BlueDevilAction("c", "bluedevil") )
 
 struct Private;
 namespace BlueDevil {
diff --git a/src/kcmodule/bluedeviladapters.cpp b/src/kcmodule/bluedeviladapters.cpp
index f2c2542..bebc3ce 100644
--- a/src/kcmodule/bluedeviladapters.cpp
+++ b/src/kcmodule/bluedeviladapters.cpp
@@ -40,7 +40,7 @@
 #include <kpluginfactory.h>
 
 K_PLUGIN_FACTORY(BlueDevilFactory, registerPlugin<KCMBlueDevilAdapters>();)
-K_EXPORT_PLUGIN(BlueDevilFactory("bluedeviladapters"))
+K_EXPORT_PLUGIN(BlueDevilFactory("bluedeviladapters", "bluedevil"))
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
diff --git a/src/kcmodule/bluedevildevices.cpp b/src/kcmodule/bluedevildevices.cpp
index 29be86a..143106f 100644
--- a/src/kcmodule/bluedevildevices.cpp
+++ b/src/kcmodule/bluedevildevices.cpp
@@ -48,7 +48,7 @@
 #include <klocalizedstring.h>
 
 K_PLUGIN_FACTORY(BlueDevilFactory, registerPlugin<KCMBlueDevilDevices>();)
-K_EXPORT_PLUGIN(BlueDevilFactory("bluedevildevices"))
+K_EXPORT_PLUGIN(BlueDevilFactory("bluedevildevices", "bluedevil"))
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
diff --git a/src/kcmodule/bluedeviltransfer.cpp b/src/kcmodule/bluedeviltransfer.cpp
index fd8170d..8ac42eb 100644
--- a/src/kcmodule/bluedeviltransfer.cpp
+++ b/src/kcmodule/bluedeviltransfer.cpp
@@ -35,7 +35,7 @@
 #include <kconfigdialogmanager.h>
 
 K_PLUGIN_FACTORY(BlueDevilFactory, registerPlugin<KCMBlueDevilTransfer>();)
-K_EXPORT_PLUGIN(BlueDevilFactory("bluedeviltransfer"))
+K_EXPORT_PLUGIN(BlueDevilFactory("bluedeviltransfer", "bluedevil"))
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
diff --git a/src/kio/bluetooth/kiobluetooth.cpp b/src/kio/bluetooth/kiobluetooth.cpp
index 668a712..fac1ea4 100644
--- a/src/kio/bluetooth/kiobluetooth.cpp
+++ b/src/kio/bluetooth/kiobluetooth.cpp
@@ -39,7 +39,7 @@ using namespace BlueDevil;
 
 extern "C" int KDE_EXPORT kdemain(int argc, char **argv)
 {
-    KAboutData about("kiobluetooth", "kiobluetooth", ki18n("kiobluetooth"), 0);
+    KAboutData about("kiobluetooth", "bluedevil", ki18n("kiobluetooth"), 0);
     KCmdLineArgs::init(&about);
 
     KApplication app;
-- 
1.7.4.1

